tests/test_tzinfo.py

```diff
@@ -163,6 +164,13 @@ def test_comparison(self):
         self.assertFalse(tz <= SMALLEST)
         self.assertTrue(tz >= SMALLEST)
 
+        # offset based comparion tests for tzinfo derived classes like datetime.timezone.
+        utcdatetime = self.DT.replace(tzinfo=timezone.utc)
+        self.assertTrue(tz == utcdatetime.tzinfo)
+        estdatetime = self.DT.replace(tzinfo=timezone(-timedelta(hours=5)))
+        self.assertTrue(self.EST == estdatetime.tzinfo)
+        self.assertTrue(tz > estdatetime.tzinfo)
+
```

Comment:
I think we should add tests to confirm that comparisons with `zoneinfo.ZoneInfo("Europe/London")` or similar don't succeed.

Reply:
Test added for `zoneinfo.ZoneInfo` would check if it return false. As, `NotImplemented` would not throw error. there would be identity check eventually as `__eq__` is not implemented for right-side and left-side

Reply:
I think by returning `NotImplemented` then equality will return `False` and ordering operators will raise a `TypeError`. I see you added an equality test, maybe add a test that ordering throws `TypeError` and then we're good here?

Reply:
Got it. Thank you so much @davidhewitt .